roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2

```diff
@@ -22,10 +22,4 @@ crd_paths:
 {% endfor %}
 {% endif %}
 {% if ocp_versions|length < 1 %}
-ocp_versions: []
-{% else %}
-ocp_versions:
-{% for ocp_version in ocp_versions %}
-  - {{ ocp_version }}
-{% endfor %}
-{% endif %}
+ocp_versions: {{ ocp_versions }}
```

Comment:
@dirgim endif missing

Reply:
Good catch, `if` wasn't supposed to be opened at all.